### ISSUE TITLE
intro-weave-gitops page is not opening at all it shows

### ISSUE THREAD

**Issue by zameer712** [weaveworks/weave-gitops#4194]
https://[docs.gitops.weave.works/docs/intro-weave-gitops/](https://docs.gitops.weave.works/docs/intro-weave-gitops/)

This XML file does not appear to have any style information associated with it. The document tree is shown below.

<Error>
<Code>UserProjectAccountProblem</Code>
<Message>The project to be billed is associated with a delinquent billing account.</Message>
</Error>

![image](https://github.com/weaveworks/weave-gitops/assets/102033066/0cae8bc0-835f-4637-ae0f-63091241a355)

**Comment by HannesHil** [comment]
Considering the recent news https://thenewstack.io/end-of-an-era-weaveworks-closes-shop-amid-cloud-native-turbulence/ its no wonder the account hasnt been paid for. 

If anybody knows a good UI alternative i would appreciate a pointer. :)

**Comment by zameer712** [comment]
@HannesHil yeah hannes you are right @stefanprodan please help us here if their is anyother for an Flux UI

**Comment by stefanprodan** [comment]
See https://fluxcd.io/blog/2024/02/introducing-capacitor/

**Comment by Igorgro** [comment]
What will happen with weave-gitops? It looks much more mature that capacitor.

**Comment by Btijmen** [comment]
> What will happen with weave-gitops? It looks much more mature that capacitor.

Since it's OpenSource, anybody can fork it and continue support for it. However, that would realistically need a sponsor with some money, so it remains to be seen if any of the companies using it are willing to back it.

**Comment by casibbald** [comment]
Weave Gitops has transitioned to a community project and a release supporting flux v2.4.0 is about to come out
https://docs.gitops.weaveworks.org/docs/intro-weave-gitops/
